src/site/_includes/components/navigation/macro.njk

```diff
@@ -0,0 +1,15 @@
+{% macro navigation(params) %}
+<nav class="js-guidance-navigation guidance-navigation" aria-label="Primary">
+  <div class="guidance-width-container">
+  <ul class="guidance-navigation__list">
+    {% for item in params.items %}
+    <li class="guidance-navigation__list-item{% if params.activeUrl == item.url%} guidance-navigation__list-item--current{% endif %}" href="{{item.url}}">
+      <a class="guidance-navigation__link govuk-link govuk-link--no-visited-state govuk-link--no-underline govuk-!-font-weight-bold">
```

Comment:
Don't see `guidance-navigation__link` being used, did you mean to use it?

Might be able to get rid of `govuk-link--no-visited-state` since the colours are being overridden anyway.